docs/codelabs/plz_query.md

```diff
@@ -156,18 +156,18 @@ $ plz query revdeps --exclude //src/greetings:greetings_test --level=-1 //third_
 Total time: 40ms real, 0s compute.
 ```
 
-As you can see, we excluded the test from earlier so `plz test` didn't run it. We can also exclude this in the query:
+As you can see, we excluded the test from earlier so `plz test` didn't run it. We can also exclude this on the test 
+command:
 ```
-$ plz query revdeps --exclude //src/greetings:greetings_test --level=-1 //third_party/go:_assert-assert#download | plz test -
+$ plz query revdeps --level=-1 //third_party/go:_assert-assert#download | plz test --exclude //src/greetings:greetings_test -
 0 test targets and 0 tests run; 0 passed.
 Total time: 40ms real, 0s compute.
 ```
 
 ### Including based on label
 
-Targets can be labeled in Please. Most of the built in rules apply some basic labels. The language rules apply language 
-labels e.g. the go rules apply the `go` label to their targets. These can be very usful to run all tests for a given 
-language:
+Targets can be labeled in Please. Most of the built in rules apply some basic labels e.g. the go rules apply the `go` 
```

Comment:
needs a hyphen and a comma, "Most of the built-in rules apply some basic labels, e.g. the go rules..."